server/cache/cache.go

```diff
@@ -31,6 +31,11 @@ type OIDCState struct {
 	ReturnURL string `json:"returnURL"`
 }
 
+type ClusterConnectionState struct {
```

Comment:
how about renaming this to `ClusterInfo`?

Reply:
sure, done